src/python/pants/backend/python/macros/poetry_requirements_test.py

```diff
@@ -28,6 +30,23 @@
 from pants.engine.target import Targets
 from pants.testutil.rule_runner import QueryRule, RuleRunner
 
+PyprojectAttr = TypedDict(
```

Comment:
This seems a duplicate of `poetry_requirements.py`. Can you instead import this type?